libraries/botframework-connector/package.json

```diff
@@ -22,6 +22,7 @@
     "@azure/ms-rest-js": "1.2.6",
     "@types/jsonwebtoken": "7.2.8",
     "@types/node": "^10.12.18",
+    "adal-node": "^0.2.1",
```

Comment:
@carlosscastro, can you explain in a reply to this comment why you decided to use `"adal-node"`? We talked in person, but I think this would be a good spot to sort of highlight why you went with this library instead of one of the other eight packages?

https://www.npmjs.com/~azuread

Reply:
For starters, adal-node supports certificate auth and pretty much all client credential flows while adal-js for example only supports password based auth client flow. I also double checked with @stevenic and he was fine with introducing this dependency in the connector.